### PR TITLE
Lab4

### PR DESCRIPTION
**Pull request by afonyaa**

### CODE REVIEW

File: terraform/github/variables.tf [hunk 1]
```diff
@@ -0,0 +1,3 @@
+variable "github_token" {
+  type = string
+}
```
Comment (AKurmazov) [inline terraform/github/variables.tf:3]: nit: You can also specify a variable to be sensitive, and its value will be hidden when it is prompted

**Comment by AKurmazov** [comment]
Good job 👍

**Comment by none-word** [comment]
Well done!

**Comment by Maxkoz777** [comment]
AWS is a great choice, well done!

**Pull request closed without merging**